common/mariadb-galera/helm/templates/_helpers.tpl

```diff
@@ -17,7 +17,11 @@ metadata:
   namespace: {{ .global.Release.Namespace }}
   {{- if eq .replica "notused" }}
     {{- if and (eq .type "frontend") (or (eq .component "database") (eq .component "proxysql") (eq .component "haproxy"))}}
+      {{- if .global.Values.mariadb.migration.enabled }}
```

Comment:
such changes increase the need for the unittests ;)

Reply:
I have added 2 unit tests